contentcuration/contentcuration/frontend/shared/data/serverSync.js

```diff
@@ -336,12 +337,33 @@ async function handleChanges(changes) {
   }
 }
 
+function initializeWebsockets() {
+  // Create WebSocket connection.
+  const socket = new WebSocket('ws://localhost:8080/ws/sync_socket/12312312312123/');
```

Comment:
You'll want to think about what the protocol is here, and how that will change for dev environments vs production, since production uses `https`.